src/pretix/control/templates/pretixcontrol/order/mail_history.html

```diff
@@ -0,0 +1,33 @@
+{% extends "pretixcontrol/event/base.html" %}
+{% load i18n %}
+{% load bootstrap3 %}
+{% block title %}{% trans "Email history" %}{% endblock %}
+{% block content %}
+    <h1>{% trans "Email history" %}</h1>
+    <div class="callout callout-info">
+        <p>Includes sent custom and mass emails(<em>plugin</em>) history only.</p>
+    </div>
```

Comment:
Please use ``alert alert-info`` instead of defining a new UI pattern and use ``{% trans "…" %}`` for the text.

(Also, I'd drop the "(plugin)", I don't think its helpful)